eslint.config.js

```diff
@@ -0,0 +1,64 @@
+const js = require('@eslint/js');
+const globals = require('globals');
+
+
+const styleRules = {
+    quotes: ['error', 'single', { avoidEscape: true }],
+    'no-trailing-spaces': 'error',
+    indent: 'error',
+    'linebreak-style': ['error', 'unix'],
+    semi: ['error', 'always'],
+    'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+    'keyword-spacing': 'error',
+    'space-before-blocks': 'error',
+    'space-before-function-paren': [
+        'error',
+        { anonymous: 'always', named: 'never' },
+    ],
+    'no-mixed-spaces-and-tabs': 'error',
+    'comma-spacing': ['error', { before: false, after: true }],
+    'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+};
+
+
+module.exports = [
+    js.configs.recommended,
+    {
+        languageOptions: {
+            ecmaVersion: 2022,
+            globals: {
+                ...globals.node,
+                ...globals.browser,
+                request: 'readonly',
+                response: 'readonly',
+                expect: 'readonly',
+                describe: 'readonly',
+                it: 'readonly',
+                render: 'readonly',
+                cleanHtml: 'readonly',
+                beforeEach: 'readonly',
+                sinon: 'readonly',
```

Comment:
These look like they should be test globals only. If these are for the client-side JS tests then they can be in a new config object targeting `components/**/spec.*.js`